src/app/lib/globalStyles.js

```diff
@@ -30,17 +23,20 @@ injectGlobal`
   }
   @font-face {
     font-display: optional;
-    font-family: ReithSansNewsMedium;
+    font-family: ReithSerifNewsMedium;
     font-style: normal;
     font-weight: 600;
-    src: url('https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_Md.woff2') format('woff2'), url('https://gel.files.bbci.co.uk/r2.302/BBCReithSans_W_Md.woff') format('woff');
+    src: url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff2') format('woff2'), url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff') format('woff');
   }
+`;
+
+/* Unused fonts
```

Comment:
Can we add a comment to be sure to add the font to the `externals` array so we don't forget?